src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/resource.rb

```diff
@@ -23,6 +23,10 @@ class Resource
     # @return [Hash{String => String}] the tags attached to resources (optional).
     attr_accessor :tags
 
+    # @return [String] the kind of the resource.
+    # FIXME kind attribute was introduced because of SiteInstance model of azure_mgmt_web gem has kind attribute.

```

Comment:
@devigned I see the "kind" but it is not part of the Resource definition. azure_arm_web is adding "kind" in the [resource ](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-web/2015-08-01/swagger/service.json#L21441)itself. Isn't that different?